### ISSUE TITLE
[BUG] datetime filter with connector AthenaPostgreSQLConnector and Athena v3

### ISSUE THREAD

**Issue by RubenR1** [awslabs/aws-athena-query-federation#940]
**Describe the bug**
Datetime filters does not work since release v2022.51.4 of AFQ. Adding the logic as a select column does return the correct true and false values but the same logic as a where clause does not return any results or show any scanned values.

**To Reproduce**
Query a postgres table and add a datetime where clause.

**Connector Details (please complete the following information):**
 - Version: 2022.51.4
 - Name postres
 - Athena Query IDs 849b2b9d-a4f3-45d3-ad6b-6376b8267df4

**Additional context**
Reverting to Athena v2 has resolved the issue.

**Comment by henrymai** [comment]
Can you also provide the region for your Athena query id and also provide a query id for the working query in Athena v2.
I'm assuming that you're using the same connector for both v2 and v3, right?

**Comment by RubenR1** [comment]
Region: eu-west-1
Yes I'm using the same connector for both Athena versions.

Working query id for Athena v2: 5bcec29b-6d61-4474-900e-5139e6b802e5

**Comment by RubenR1** [comment]
Same query not working for Athena v3: 849b2b9d-a4f3-45d3-ad6b-6376b8267df4

**Comment by henrymai** [comment]
@zsvoboda

Sorry this is taking so long but the team is still working on it.

**Comment by henrymai** [comment]
Closing this ticket as a duplicate of:
https://github.com/awslabs/aws-athena-query-federation/issues/1018

Chose to unify on 1018, because it has the most comprehensive post (contains full reproduction steps).

**Comment by henrymai** [comment]
@RubenR1
I also posted a workaround to unblock you if you are blocked here: https://github.com/awslabs/aws-athena-query-federation/issues/1018#issuecomment-1425265581
